spec/features/login_signup/login_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227711193-bc79ff05-14ed-4c2a-9468-8e6b0cfa8b94.png)

**[OPTIONAL]**

- [ ] It would be great if you make your project summary more descriptive to make your project more professional :+1: